apps/calendar/build/build.js

```diff
@@ -29,11 +29,12 @@ exports.execute = function(options) {
   utils.ensureFolderExists(utils.getFile(options.STAGE_APP_DIR));
 
   dump('Will run rjs optimizer...\n');
-  var optimize = new Promise((accept, reject) => {
+  var optimizejs = new Promise((accept, reject) => {
     requirejs.optimize([config.path], accept, reject);
   });
 
-  optimize.then(() => {
+  optimizejs.then((log) => {
```

Comment:
yes.. before I had separate targets for JS/CSS that's why I renamed the `optimize` to `optimizejs`..

do you think the log is a good thing or not necessary? (looks like a `build.txt` file is also saved on the `build_stage/calendar` folder)